src/EventListener/RespondListener.php

```diff
@@ -36,7 +36,7 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
         $controllerResult = $event->getControllerResult();
         $request = $event->getRequest();
 
-        if ($controllerResult instanceof Response || !$request->attributes->get('_api_respond')) {
+        if ($controllerResult instanceof Response || !$request->attributes->getBoolean('_api_respond', true)) {
```

Comment:
Not sure that this change is necessary or we'd have to change other ones for consistency.

Reply:
These changes are not even from me, I created the pull request from the wrong point. But this is now fixed.